settings.php

```diff
@@ -51,4 +51,13 @@
     $settings->add(new admin_setting_configduration('tool_advancedreplace/logduration',
         get_string('settings:logduration', 'tool_advancedreplace'),
         get_string('settings:logduration_help', 'tool_advancedreplace'), 30, 1));
+
+    $ADMIN->add(
+        'tools',
```

Comment:
This should be moved under advancereplacefolder now. 

May also be worth moving either this or the search to make them next to each other.

Reply:
I tried to fix this, but I'm not sure I got it right.  I would like to save this for a future commit.